core/src/test/scala/org/apache/spark/sql/delta/DeltaErrorsSuite.scala

```diff
@@ -1765,10 +1765,10 @@ trait DeltaErrorsSuiteBase
            |configurations when creating the SparkSession as shown below.
            |
            |  SparkSession.builder()
-           |    .option("spark.sql.extensions", "${classOf[DeltaSparkSessionExtension].getName}")
-           |    .option("$catalogImplConfig", "${classOf[DeltaCatalog].getName}")
+           |    .config("spark.sql.extensions", "${classOf[DeltaSparkSessionExtension].getName}")
```

Comment:
without import does the `classOf[DeltaSparkSessionExtension]` work? If it doesn't work, may be add the fully qualified class name.

Reply:
Makes sense. I updated the DeltaErrors.scala to write the canonical name